sdk/ownership/codec.go

```diff
@@ -0,0 +1,16 @@
+package ownershipsdk
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+)
+
+// ModuleCdc is the codec for the module
+var ModuleCdc = codec.New()
+
+func init() {
```

Comment:
Can we replace all init function with explicit call?

Reply:
That's following the guidelines of cosmos. Also, we need to make sure that the codec is initialized even if the backend is not yet created. We could have some sync of the messages before we initialize the backend.

We could find a way to remove the init and have a singleton pattern but that will be complex for no real advantage.

Reply:
> That's following the guidelines of cosmos

Yes, i think cosmos is not the right place to get best pracitices. 

>  Also, we need to make sure that the codec is initialized even if the backend

That's easy, we can call InitCodes inside main function. 

Liek for me `init` should be avoided at all cost, because it changes the workflow of execution. There are only few places where it should be used (like goto, or brake/continue outer loops)

Reply:
> Yes, i think cosmos is not the right place to get best pracitices

Maybe not but protobuf or gogoproto might be ;)
Anyway we could have a function that the is called on every package and make sure that this is called at the beginning of main but at the end that's similar to init.